===================================================================
RCS file: /usr/local/www/cvsroot/FreeBSD/src/gnu/usr.bin/cpio/Attic/main.c,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -p -r1.2 -r1.3
--- src/gnu/usr.bin/cpio/Attic/main.c	1995/10/26 11:12:39	1.2
+++ src/gnu/usr.bin/cpio/Attic/main.c	1997/02/25 06:11:19	1.3
@@ -459,7 +459,6 @@ main (argc, argv)
      char *argv[];
 {
   program_name = argv[0];
-  umask (0);
 
 #ifdef __FreeBSD__
   (void) setlocale (LC_ALL, "");
@@ -473,6 +472,7 @@ main (argc, argv)
 #endif
 
   process_args (argc, argv);
+  umask (0);
 
   initialize_buffers ();
 
